SPECS/systemd/systemd-bootstrap.spec

```diff
@@ -12,6 +12,7 @@ Source1:        50-security-hardening.conf
 Source2:        systemd.cfg
 Source3:        99-dhcp-en.network
 Patch0:         fix-journald-audit-logging.patch
+Patch1:         add-fsync-sysusers-passwd.patch
```

Comment:
Thanks for the patch and porting it to Mariner! Let's also add a short comment here, so it's easier to track when this patch can be removed:

```suggestion
# Can be removed once we update systemd to a version containing the following commit:
# https://github.com/systemd/systemd/commit/19193b489841a7bcccda7122ac0849cf6efe59fd
Patch1:         add-fsync-sysusers-passwd.patch
```